pkg/placement/controller.go

```diff
@@ -420,6 +449,12 @@ func (c *Controller) reconcile(ctx context.Context, key util.Key) error {
 		return nil
 	}
 
+	if err := c.updateDecisions(obj); err != nil {
+		utilruntime.HandleError(fmt.Errorf("error reconciling object: %s", err))
```

Comment:
Is it really true that no error possible here is worth a retry? The comment on `updateDecisions` should be explicit about this.